dictionaries/en-common-misspellings/src/excludeWords.txt

```diff
@@ -0,0 +1,4 @@
+# These are words that should be excluded from ALL suggestions
+# cspell:ignoreRegExp /^.*(#|$)/gm
+programing
+interpretor
```

Comment:
My understanding is that you found typos in their words.csv, right?

So you are excluding them here

Reply:
Yes.